gamebook/hero.py

```diff
@@ -0,0 +1,46 @@
+from gamebook.creature import Creature
+import random
+
+
+class Hero(Creature):
+    def __init__(self, traits):
+        self.traits = traits
+        self.is_alive = True
+
+    def attack(self, trait_name):
```

Comment:
How does this work with trait? it's kind of confusing...

Reply:
Also, why isn't this a trait obj, and just a trait name?

Reply:
i will add a docstring explaining the logic behind this function, it is like what you will do playing an actual gamebook of this kind

Reply:
> Also, why isn't this a trait obj, and just a trait name?

i thought it will be more aesthetic and better to use a string instead of the actual trait while implementing the function when building a game, do you think i should change it then?

Reply:
I think objects are better because they hold the exact 1:1 match with what you want to call, strings can change overtime and this will be broken.